crates/uv/src/commands/project/run.rs

```diff
@@ -25,8 +26,7 @@ use crate::settings::ResolverInstallerSettings;
 pub(crate) async fn run(
     extras: ExtrasSpecification,
     dev: bool,
-    target: Option<String>,
-    mut args: Vec<OsString>,
+    command: ExternalCommand,
```

Comment:
We split this into `target` and `args` late so we can avoid borrowing, cloning, or otherwise painfully popping the first item from the vector.